corehq/apps/aggregate_ucrs/tests/data/table_definitions/basic_aggregate_definition.yml

```diff
@@ -0,0 +1,25 @@
+domain: agg
+table_id: pregnancies_with_forms
+display_name: Pregnancy Data With Form Aggregates
+primary_table:
+  data_source_id: 550c3cd432d931387e75e8506b5caf9e  # the data source ID of e.g. the 'pregnancy_cases' table
+  key_column: doc_id
+  columns:
+    - column_id: name
```

Comment:
agreed - that type of thing has also been on my mind. being able to add more syntactic sugar to the spec format is actually one of the reasons I added an abstraction barrier between serialization and models that i forgot to bring up (something @snopoke was asking about).

to me the value / decision also depends on how we expect users to interact / create these things (also raised by @snopoke). It may be the case that we can build a UI that makes the verbosity less of an issue.